modules/guides/pages/javascript-udfs/creating-javascript-udf.adoc

```diff
@@ -0,0 +1,56 @@
+= Creating a Javascript UDF
+:description: Using the Couchbase Query Workbench to create a user-defined function written in Javascript
+:page-pagination: 
+:page-edition: Enterprise Edition
+:page-topic-type: guide
+:page-toclevels: 2
+
+[abstract]
+{description}
+
+== Introduction
+
+We're going to introduce a number of new concepts in this example:
```

Comment:
Well, to be clear, how-to guides don't shy away from describing concepts completely. Otherwise users would be flipping back and forth between the different pages to understand what they're doing. It's more the implicit thought behind the wording that I'm talking about: the concepts should be _introduced_ in a conceptual topic, and that topic should be linked from the how-to guide. Even if the how-to guide reiterates the concepts in brief.

Reply:
Note that the conceptual guide should be covered by DOC-9256.